source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -251,17 +251,17 @@ To begin development, MedcoMD engineers generate a locally-managed master key:
         import java.security.SecureRandom;
 
         public class CreateMasterKeyFile {
-          public static void main(final String[] args) {
+            public static void main(String[] args) {
 
-            final byte[] localMasterKey = new byte[96];
-            new SecureRandom().nextBytes(localMasterKey);
+                byte[] localMasterKey = new byte[96];
+                new SecureRandom().nextBytes(localMasterKey);
 
-            try (FileOutputStream stream = new FileOutputStream("master-key.txt")) {
-              stream.write(localMasterKey);
-            } catch (IOException e)  {
-              e.printStackTrace();
+                try (FileOutputStream stream = new FileOutputStream("master-key.txt")) {
+                    stream.write(localMasterKey);
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
Same comment as above.  Consider removing the catch clause

Reply:
Will fix